curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md

```diff
@@ -0,0 +1,168 @@
+---
+id: 62bb4009e3458a128ff57d5d
+title: Step 68
+challengeType: 0
+dashedName: step-68
+---
+
+# --description--
+
+One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` tag as a child of the `head` element. Set its `charset` attribute to:
+
+`UTF-8`
```

Comment:
```suggestion
One more thing. You should allow people to use their native language. Tell the browser to encode multiple languages by adding a `meta` tag as a child of the `head` element. Set its `charset` attribute to `UTF-8`.
```

This seems weird on a separate line.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
It's but we had issues with code block at the end of the sentence.

bbsmooth recommend new line with the value in it, to support mobile users accessibility.

Reply:
I don't like it on its own line either - and would vote to put it back on the same line. For some context, we had [some discussion on this](https://github.com/freeCodeCamp/freeCodeCamp/issues/46052) and made some changes already. The conclusion was to do this for the first few steps of this project.